doc/en/user/source/installation/docker.rst

```diff
@@ -0,0 +1,91 @@
+.. _installation_docker:
+
+Docker Container
+================
+
+Geoserver is also packaged as a Docker Container.  For more details, see the `Geoserver Docker Container Project <https://github.com/geoserver/docker>`_.
+
+Quick Start
+-----------
+
+This will run the container with the data directory shipped with the container (which is typically an empty directory).
+
+#. Make sure you have `Docker <https://www.docker.com/>`_ installed.
+#. Download the container
+
+   .. code-block:: 
+    
+      docker pull docker.osgeo.org/geoserver:2.21.1
```

Comment:
Would like to figure out a way for these commands to be updated with the version number; not sure we can do so as part of a code block.

How does the following look:
```
#. Download the container

   > docker pull docker.osgeo.org/geoserver:|release|
```